infinispan-remote/pom.xml

```diff
@@ -24,11 +24,13 @@
 
     <properties>
         <jdkTargetForProcessor>1.8</jdkTargetForProcessor>
-        <!-- Skipping enforcer plug-in as WildFly / Infinispan Server artifact
-             causes several dependency convergence issues which are beyond our control -->
-        <enforcer.skip>true</enforcer.skip>
+
         <infinispanServerName>infinispan-server</infinispanServerName>
         <infinispan-server.home>${project.build.directory}/${infinispanServerName}</infinispan-server.home>
+
+        <!-- Skipping enforcer dependency convergence check as WildFly / Infinispan Server artifact
+             causes several dependency convergence issues which are beyond our control -->
+        <enforcer.dependencyconvergence.skip>true</enforcer.dependencyconvergence.skip>
```

Comment:
The property now inhibits only dependencyconvergence not the full enforcer. I steal the pattern from Hibernate Search.

Reply:
Got it, nice one

Reply:
I usually keep this changes in a separate commit though:
1. they could be applied as a separate issue
2. It make it easier for the reviewer to figure out why something has changed (because it is described in the commit message)
3. It's not required to solve the issue

Anyway, don't change it. It is a nice example of what I consider when I need to decide if something should be in a separate commit (we had this conversation in the past).